components/core/Profile.js

```diff
@@ -8,40 +8,125 @@ import { ProcessedText } from "~/components/system/components/Typography";
 import SlatePreviewBlocks from "~/components/core/SlatePreviewBlock";
 
 const STYLES_PROFILE = css`
-  text-align: center;
   width: 100%;
-  margin-top: 16px;
+  padding: 24px 24px 0px 24px;
+`;
+
+const STYLES_PROFILE_INFO = css`
+  display: flex;
+  line-height: 1.3;
+  margin: 0 auto;
+  width: 50%;
```

Comment:
Can you make this:
``` 
width: 100%;
max-width: ${Constants.sizes.desktop}px;
```
That's the width and max width we have on the slate blocks that make up teh rest of the profile page

Reply:
cool! will do